Please dont make contributors hunt for instructions

philsturgeon opened the pull request:
Nice easy tips for contributors getting started, who might not know where to look otherwise.

Commit:
Hunting through travis.yml isnt ideal

fenollp commented:
FWIW to me .travis,yml is the first place to look at when trying to understand how to build & test a project, given that
* it's its purpose
* it usually works
* there is usually not much to read there

What is the first place you look at? I'm interested as I should probably update some of my projects accordingly

philsturgeon commented:
That's cool if you've heard of Travis and know think to look at the config file. 👍🏼

If they switch away from Travis to some other thing are you gonna know where that config file lives too?

localheinz commented on README.md:
```diff
@@ -91,10 +91,12 @@ API-Flow is one of the main components of [Console.REST](https://github.com/luck
 ## Contributing
 
-PRs are welcomed!
+Read [CONTRIBUTING.md][] for more tips on testing and contributing.
```
How about adding a link here, @philsturgeon?

Or is this equivalent with `[CONTRIBUTING.md](CONTRIBUTING.md)`?

philsturgeon replied:
Fixed! Thanks, I was using some random syntax that clearly isn't GFM

localheinz commented on README.md:
```diff
@@ -91,10 +91,12 @@ API-Flow is one of the main components of [Console.REST](https://github.com/luck
 ## Contributing
 
-PRs are welcomed!
+Read [CONTRIBUTING.md](CONTRIBUTING.md) for more tips on testing and contributing.
```
👍

localheinz commented:
@fenollp 

Also see https://github.com/luckymarmot/API-Flow/community! 🤓

Commit:
Local links work well for GitHub READMEs

A few forks were updating these links and causing conflicts, so lets just use local links :D
